src/SFA.DAS.PR.Application.UnitTests/EmployerRelationships/Queries/GetProviderEmployerRelationship/GetProviderEmployerRelationshipQueryHandlerTests.cs

```diff
@@ -0,0 +1,167 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using Moq;
+using SFA.DAS.PR.Application.EmployerRelationships.Queries.GetProviderEmployerRelationship;
+using SFA.DAS.PR.Application.Mediatr.Responses;
+using SFA.DAS.PR.Data.UnitTests.Setup;
+using SFA.DAS.PR.Domain.Entities;
+using SFA.DAS.PR.Domain.Extensions;
+using SFA.DAS.PR.Domain.Interfaces;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.PR.Application.UnitTests.EmployerRelationships.Queries.GetProviderEmployerRelationship;
+
+public class GetProviderEmployerRelationshipQueryHandlerTests
+{
+    private readonly CancellationToken cancellationToken = CancellationToken.None;
+
+    [Test]
+    [RecursiveMoqAutoData]
+    public async Task Handle_GetProviderEmployerRelationship_Returns_GetProviderEmployerRelationshipQueryResult(
```

Comment:
A test for what the response looks like when there is a relationship but no permissions is missing.

Reply:
@DevenKShah That scenario is covered in one of the tests.